testregistry.textproto

```diff
@@ -930,6 +930,12 @@ test: {
   readme: "https://github.com/openconfig/featureprofiles/blob/main/feature/interface/staticarp/ate_tests/static_arp_test/README.md"
   exec: " "
 }
+test: {
+  id: "TE-1.2"
```

Comment:
Add to DP list of tests.  (dataplane).

TE is for traffic engineering features.

Reply:
done